solr/core/src/test-files/solr/collection1/conf/solrconfig-caching.xml

```diff
@@ -21,19 +21,16 @@
   <schemaFactory class="ClassicIndexSchemaFactory"/>
   <query>
     <cache name="lfuCacheDecayFalse"
```

Comment:
I think we shouldn't be changing this file. It's specific for the LFUCache testing, so we should leave it as is.

Reply:
Ok, I reverted the changes

Reply:
@trdillon, I created https://github.com/apache/lucene-solr/pull/2174, to remove this file completely from master, it's not being used since the deprecated cache implementations and their tests have been removed.